overrides.nix

```diff
@@ -619,6 +625,20 @@ self: super:
     }
   );
 
+  inquirer = super.inquirer.overridePythonAttrs (
+    old: {
+      preBuild = old.preBuild or "" + ''
+        substituteInPlace setup.py --replace 'version = "3.0.0"' 'version = "${old.version}"'
+      '';
+      preConfigure = (old.preConfigure or "") + ''cat << EOF > requirements.txt
+blessed==1.19.0
+readchar==2.0.1
+python-editor==1.0.4
+EOF
+      '';
+    }
+  );
+
```

Comment:
What's a better way to add this file. inquirer's setup.py calls a function that reads the requirements.txt file.

perhaps add a list of dependencies manually is better, avoiding requirements.txt altogether?